backend/maelstro/main.py

```diff
@@ -139,6 +139,19 @@ def get_layers(src_name: str, uuid: str) -> list[dict[str, str]]:
         return meta.get_ogc_geoserver_layers()
 
 
+@app.get("/involved_resources")
+def get_involved_resources(
+    src_name: str,
+    dst_name: str,
```

Comment:
we don't need the destination name, do we ?

Reply:
I found interesting having the destination name in a "preview_copy" response, and it is not much more expensive.

Reply:
well, I don't mind. I was just very puzzled to find no processing at all on the destination_name.
However, something much more important just came to my mind: The preview_entrypoint should also list the xslt queue to be applied with their descriptions. https://github.com/georchestra/maelstro/pull/36/files/98729333bf2e8af58d08df3740dbf3be6dbc4e21#diff-047d6066b8e27433cfcb48f4fc320d444e13707912929ed2fc9d66fb2bcdd552R83
And here we need the destination name. What do you think about that ?

Reply:
Yes, could be interesting to return list of XSLT transformations, but cannot do it in this PR yet.

Reply:
ok, I'll keep it in mind after merge of PR #36